Project-2/README.md

```diff
@@ -10,42 +10,134 @@ Tip #4: Work together _as a team_.
 ```
-  (a)  PP that says that x is a proper part of y
+  (a)  PP that says that x is a proper part of y 
+- P means x is a part of y
+- ~P means x is not a part of y
+- P^_ means y is a part of x
+  
+- A proper part is like a one-sided (subsumption) relationship.
+- Finn, Giacomo, Ali: PP:= ∃P.~P OR PP:= ∃(P⊓~P^_)
+
   (b)  iPP that says that y is a proper part of x
+ - Finn, Giacomo, Ali: iPP:= PP^_
+ 
   (c)  iP that says that y has x as part 
+iP := P^_
+
   (d)  O that says that x overlaps y
+ - Pulled from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1175958/: "r1 overlap r2 = [definition] for some r, r part_of r1 and r part_of r2."
+0 := ∃P.P^_
+ - Could ∃(P⊓P^_) also work for the above?
+ 
   (e)  D that says that x and y are disjoint 
-```
+D:= 0^_
+
+``` 
 
 2. Use your axioms from question 1 as the basis of an ALCI T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+- These slides https://www.inf.unibz.it/~calvanese/teaching/16-17-odbs/lecture-notes/KRO-6-alc.pdf are the best
+- https://www.researchgate.net/publication/326360368_Automated_noninvasive_epithelial_cell_counting_in_phase_contrast_microscopy_images_with_automated_parameter_selection/figures?lo=1 Figure 3 here helps me think about parthood for the purposes of my T box
+
+Instances: {Hand, Severed_Hand, Handless_Darcy, Darcy_2Hands}
+Concepts: {Limb, Person}
+Relations: {P, PP, P¯, PP¯, 0, D}
+
+T Box: inclusive Terms
+ 
+
+A box: membership Assertions
+
+Person: {Handless_Darcy, Darcy_2Hands}
+Limb: {Hand, Severed_Hand}
+
+
 
 3. Translate the following first-order logic axioms into ALC: 
 ```
 (a) ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+
 (b) ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+
 (c) ∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y))))
+- ALC does not include conditionals or biconditionals in its syntax, so first I'm going to re-express the given formula without them:
+∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y)))) ≡ ∀y(R(x,y) ∨ ¬(∃x(R(y,x))) ∧ ∀y(R(x,y)∨ ¬A(y))
+
+
 (d) (∀y)(R(x, y) → A(y)) ∧ (∃y)(R(x, y) ∧ B(y))
+(∀y)(R(x, y) → A(y)) ∧ (∃y)(R(x, y) ∧ B(y)) ≡ (∀y)(R(x, y) ∨ ¬A(y)) ∧ (∃y)(R(x, y) ∧ B(y))
+
 ```
 4. Provide an interpretation I<sub>1</sub> for ALC and an interpretation I<sub>2</sub> for ALCN - each distinct from any interpretation covered in class so far - and construct a bisimulation that demonstrates ALCN is more expressive than ALC. Use the [mermaid syntax](https://github.com/mermaid-js/mermaid) of markdown to provide a graphical representation of your work. 
 
+- ALCN is a syntax identical to ALC with "...unqualified number restrictions... concepts of the form (≤nr.T) and (≥nr.T)" (Baader et. al., 63)
+- So, in the textbook, once a set contains more or less than 1 relation, it can no longer be expressed in ALC.
+- This looks like (≤1 r).
```

Comment:
This is correct, but I was hoping to see an interpretation using bisimulation to justify why ALC cannot express ALCN terms. One might think if we're clever enough using just the resources of ALC, we might be able to describe cardinality restrictions. Bisimulation is a way to show that we shouldn't waste our time trying, since we can prove there is no such clever construction.